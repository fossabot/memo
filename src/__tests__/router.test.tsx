import React from 'react';
import { shallow } from 'enzyme';
import Router from '../router';
import 'nino-cli/scripts/setup';

describe('Router', () => {
  it('render correctly', () => {
    expect(shallow(<Router />)).toMatchSnapshot();
  });
});
