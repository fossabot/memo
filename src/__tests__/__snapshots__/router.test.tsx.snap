// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router render correctly 1`] = `
<HashRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/new"
    />
    <Route
      component={[Function]}
      key="b07b876e6050e333988404e268921906"
      path="/b07b876e6050e333988404e268921906"
    />
    <Route
      component={[Function]}
      key="c9abecfb05549080298f37daea8de46f"
      path="/c9abecfb05549080298f37daea8de46f"
    />
    <Route
      component={[Function]}
      key="e6342ea80e1b9da29570d49af1daaaeb"
      path="/e6342ea80e1b9da29570d49af1daaaeb"
    />
  </Switch>
</HashRouter>
`;
